solidity/test/Oracle_test.js

```diff
@@ -266,6 +265,15 @@ contract('Oracle', () => {
           })
         })
       })
+
+      context('lies about amount sent', () => {
```

Comment:
The wording of the context and the test seem at odds? Can the context be deleted as it doesn't require any extra setup?